src/retinanet.py

```diff
@@ -142,9 +142,13 @@ def forward(self, x):
     def _make_head(self, out_planes):
         layers = []
         for _ in range(4):
-            layers.append(nn.Conv2d(256, 256, kernel_size=3, stride=1, padding=1))
+            head_layer = nn.Conv2d(256, 256, kernel_size=3, stride=1, padding=1)
+            head_layer.__name__ = 'head_layer'
```

Comment:
@taraspiotr it doesn't feel super clean to use double underscore to pass a layer name. Is there any "clean" way of doing it in pytorch?

Since we are doing it simply to "brand" the layer to know which layers should we initialize with retine initialization can we simply use head_layer.name = 'head_layer' ?

Reply:
@jakubczakon yes, I was thinking about that as well and it will be cleaner.